features/_your_results/salary_below_minimum_threshold.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Employer contributions do not increase when user's salary is less than the minimum threshold
+  In order to understand that there is no requirement for employer contributions to increase over time
+  As an employee earning under £5876
+  I need to see employer contributions stay static
+
+  Scenario Outline: For monthly, 4-weekly and weekly salary frequencies
+    Given my "<salary>" "<salary_frequency>" is below the minimum threshold
+    And my employee contribution is "<my_contribution>"
+    And my employer contribution is "<employer_contribution>"
+    When I move on to the results page
+    Then I should see my employer contributions for current period as "<employer_current_period>"
+    And I should see my employer contributions for second period as "<employer_second_period>"
+    And I should see my employer contributions for second period as "<employer_third_period>"
+
+    Examples:
+        | salary | salary_frequency | my_contribution | employer_contribution | employer_current_period | employer_second_period | employer_third_period |
+        | 416.67 | per Month        | 1               | 0                     | £0.00                   | £0.00                  | £0.00                 |
```

Comment:
Happy to chat in person about this, I think we can simplify these features a little more. 
- `my_contribution` is the same and so can be written into the step rather than the table. 
- the 3 columns `employer_current_period`, `employer_second_period`, `employer_third_period` are all identical. I think it makes it harder to read what is being tested. Why not just have the 1 column and call it `contribution_from_employer` - or something that makes sense..